Dockerfile

```
@@ -126,6 +130,14 @@ RUN git clone -b master https://github.com/jesims/circleci-tools.git \
 ENV PATH=$PATH:/tmp/circleci-tools/
 RUN node -v > .node_version
 
+#-- Install cljog
+RUN wget https://raw.githubusercontent.com/axrs/cljog/${CLJOG_VERSION}/cljog \
+  && chmod u+x cljog \
+  && mv cljog /usr/bin/ \
```

Not `/usr/local/bin`?
"/usr/local/bin is for normal user programs not managed by the distribution package manager"
```suggestion
  && mv cljog /usr/local/bin/ \
```

---

👍